pkg/controller/plan/util/utils.go

```diff
@@ -20,8 +21,14 @@ func roundUp(requestedSpace, multiple int64) int64 {
 	return int64(partitions) * multiple
 }
 
-func CalculateSpaceWithOverhead(requestedSpace int64) int64 {
+func CalculateSpaceWithOverhead(requestedSpace int64, volumeMode *core.PersistentVolumeMode) int64 {
+	var overhead int
+	if *volumeMode == core.PersistentVolumeFilesystem {
+		overhead = settings.Settings.FileSystemOverhead
+	} else {
+		overhead = settings.Settings.BlockOverhead
+	}
```

Comment:
I'm not sure the block overhead should also be in percentage - for things like the encryption header on Ceph, we can use a fixed overhead instead